_data/pt-BR.yml

```
@@ -0,0 +1,294 @@
+footer:
+  notificationApp: App de notificações da Bisq
+  siteYear: © 2019 Bisq
+  ulResources: Recursos
+  liDownloads: Downloads
+  liRoadmap: Plano
+  liContribute: Contribuir
+  liMarketData: API de dados de mercado
+  joinBisq: Junte-se à Bisq
+
+homepage_content:
+  title: "<span class=\"animate1\">Exchange,</span> <span class=\"animate2\">Descentralizada.</span>"
+  intro: "A Bisq é um programa par-a-par de código aberto que permite que você compre e venda criptomoedas em troca de moedas nacionais. Nenhum registro é necessário."
+  buttonWindows: Download para Windows
+  buttonMac: Download para Mac
+  buttonDebian: Download para Debian/Ubuntu
+  buttonRedHat: Download para Red Hat/Fedora
+  buttonDownloads: Downloads
+  buttonGetBisq: Baixar Bisq
+  buttonToggle: Alternar o menu dropdown
+  downloads: Downloads
+  androidNotificationsApp: Baixar o App de Notificações para Android
+  iosNotificationApp: Baixar o aplicativo de notificações para iOS
+  getStarted: Iniciar
+  runningWindows: Parece que você está usando um Windows.
```

```suggestion
  runningWindows: Parece que você está usando um Windows
```

For consistency, as none of the other languages include a period here. This variable and the other one should be merged since they're exactly the same, but that's for another time.